nexus/examples/qmcpack/rsqmc_misc/estimators/iron_ldaU_dmc.py

```diff
@@ -0,0 +1,220 @@
+    walkers_per_rank     = 256,
+    samples              = 1e5,
+    blocks               = 10,
+    minwalkers           = 0.3,
+    corrections          = [],
+    dependencies         = (p2q,'orbitals'),
+    **qmc_shared
+    )
+
+J3_rcut = system.structure.rwigner()
+optJ123 = generate_qmcpack(
+    identifier           = 'optJ123',
+    path                 = basepath + 'optJ123',
+    job                  = job(cores=16,threads=4,app='qmcpack'),
+    system               = system,
+    twistnum             = 0,
+    qmc                  = 'opt',
+    J2                   = True,
+    J3                   = True,
+    J3_rcut              = J3_rcut,
+    minmethod            = 'oneshift',
+    warmupsteps          = 10,
+    init_cycles          = 0,
+    cycles               = 10,
+    walkers_per_rank     = 256,
+    samples              = 1e5,
+    blocks               = 10,
+    minwalkers           = 0.5,
+    corrections          = [],
+    dependencies         = [(p2q,'orbitals'), (optJ12,'jastrow')],
+    **qmc_shared
+    )
+
+
+#===== Spin density estimator =====
+from qmcpack_input import spindensity
+sdens = spindensity(
+    dr = (0.03, 0.03, 0.03),   # Bohr units
+    #grid = (100,100,100),  # Alternative to dr, an NxNxN grid can be specified
+    )
+
+#===== Energy density =====
+from qmcpack_input import generate_energydensity
+edens = generate_energydensity(
+    coord = 'cartesian',
+    grid  = (100, 100, 100),
+)
+
+#===== Momentum distribution =====
+from qmcpack_input import momentumdistribution
+momdist = momentumdistribution(
+    samples = 40,
+    kmax    = 8.0,
+)
+
+#===== One body density matrix =====
+from qmcpack_input import onebodydensitymatrices, sposet
+nbnd = nscf.input.system.nbnd   # Total number of bands solved in KS-DFT for each spin channel (occupied + virtual)
+# For magnetic systems such as iron, the spin-up and spin-down 1RDMs are distinct.
+# Currently, the 1RDM estimator can only project onto a single specified basis.
+# The estimator below uses the spin-up orbitals as the basis in the 1RDM for both spin channels.
+# To get the proper spin-down 1RDM, a seperate QMC run with spin-down basis (spindataset = 1) is needed at the moment.
+dm_est = onebodydensitymatrices(
+    type          = 'OneBodyDensityMatrices',
+    energy_matrix = False,
+    check_overlap = False,
+    integrator    = 'density',
+    evaluator     = 'matrix',
+    samples       = 10,
+    basis         = sposet(type = 'bspline', size = nbnd, spindataset = 0), # Uses the spin up channel (0) KS-DFT orbitals as the DM basis
+)
+
+qmc = generate_qmcpack(
+    #skip_submit          = True,
+    identifier           = 'qmc',
+    path                 = basepath + 'qmc',
+    job                  = job(cores=12,threads=4,app='qmcpack'),
+    system               = system,
+    twistnum             = 0,
+    estimators           = [sdens, edens, momdist, dm_est],  # Requested estimators. These will be included in all QMC series.
```

Comment:
Do these not appear in an `<estimators>` XML block in the input file (batched drivers)?  If not, then I need to make an update.

Reply:
They do properly appear in a dedicated `<estimators>` XML block. My initial comment here tried to convey that all estimators will be run in all QMC series. Slightly updated the comment.